pwa/src/components/applications/applicationForm.tsx

```diff
@@ -130,6 +130,14 @@ export const ApplicationForm: React.FC<ApplicationFormProps> = ({id}) => {
           title={title}
           cardHeader={function () {
             return (<>
+              <button
```

Comment:
**This feedback goes for every file where you've added the _Help_ button.**

@MWest2020 created a new _Help_ modal, including dynamic loading of the documentation from ReadTheDocs. We want to use this new modal within all these forms. To include this we have to do a couple of things (in each file).

**1. Create a `[document, setDocument]` state to store the documentation after the API call.**

```
const [documentation, setDocumentation] = React.useState<string>(null)
```

**2. GET the documentation using the API Service**

```
const handleSetDocumentation = (): void => {
    API.Documentation.get()
      .then((res) => {
        setDocumentation(res.data.content);
      })
      .catch((err) => {
        throw new Error("GET Documentation error: " + err);
      });
  };
```

We of course need to actually run this function at some point. You can do this within a `React.useEffect()`, preferably within an existing one. 

E.g. in `ApplicationForm` this would look like:

```
React.useEffect(() => {
    id && handleSetApplications()
    handleSetDocumentation() // we added this
  }, [API, id])
```

**3. Lastly, we need to include the actual modal**

At the same location as the previous _Help_ button, we can now add the Modal, like so:

```
<button
                className="utrecht-link button-no-style"
                data-bs-toggle="modal"
                data-bs-target="#helpModal"
                onClick={(e) => e.preventDefault()}
              >
                <Modal
                  title="Application Documentation"
                  id="helpModal"
                  body={() => (
                    <div dangerouslySetInnerHTML={{ __html: documentation }} />
                  )}
                />
```

Note: you (probably) still need to include the `Modal` component!